samples/apply-dictionary-renderer-to-graphics-overlay/src/main/java/com/esri/arcgismaps/sample/applydictionaryrenderertographicsoverlay/screens/ApplyDictionaryRendererToGraphicsOverlayScreen.kt

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.applydictionaryrenderertographicsoverlay.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.applydictionaryrenderertographicsoverlay.components.ApplyDictionaryRendererToGraphicsOverlayViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import androidx.compose.ui.Modifier
+
+/**
+ * Screen composable that displays a SceneView and binds to the [ApplyDictionaryRendererToGraphicsOverlayViewModel].
+ * The SceneView shows graphics styled with a DictionaryRenderer loaded from a web style and uses a Camera
+ * provided by the ViewModel to set the initial viewpoint.
+ */
+@Composable
+fun ApplyDictionaryRendererToGraphicsOverlayScreen(sampleName: String) {
+    val viewModel: ApplyDictionaryRendererToGraphicsOverlayViewModel = viewModel()
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = { padding ->
+            Column(modifier = Modifier
+                .fillMaxSize()
+                .padding(padding)) {
+
+                // The SceneView is a composable provided by the toolkit. All ArcGIS objects come
+                // from the ViewModel (scene, overlays, proxy) — the composable only renders them.
+                // We supply the camera from the ViewModel so the SceneView uses the camera as the
+                // initial viewpoint once the ViewModel has calculated it.
+                SceneView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISScene = viewModel.arcGISScene,
+                    sceneViewProxy = viewModel.sceneViewProxy,
+                    graphicsOverlays = listOf(viewModel.graphicsOverlay)
+                )
+
+                // Additional sample UI (controls) could go here. For this sample we keep the UI minimal.
+            }
```

Comment:
- Suggestion to remove the `Column` with one child, and display the `SceneView` as the root content for the Scaffold. 

- Suggestion to update comment for the SceneView itself, and remove invalid comment below. 

```suggestion
            // Composable SceneView that renders graphics styled with a DictionaryRenderer
            // using configured ArcGIS objects from the ViewModel (scene, overlays, proxy).
            SceneView(
                modifier = Modifier
                    .fillMaxSize()
                    .padding(padding),
                arcGISScene = viewModel.arcGISScene,
                sceneViewProxy = viewModel.sceneViewProxy,
                graphicsOverlays = listOf(viewModel.graphicsOverlay)
            )
```